pandas/tests/io/test_parquet.py

```diff
@@ -828,6 +828,35 @@ def test_additional_extension_types(self, pa):
         )
         check_round_trip(df, pa)
 
+    @td.skip_if_no("pyarrow", min_version="0.15.1.dev")
+    def test_use_nullable_dtypes(self, pa):
```

Comment:
need a test that this fails on fastparquet

Reply:
Test added